### PR TITLE
Fix/pr template

### PR DESCRIPTION
**Pull request by samajammin**
Move PR template file to the correct location (it's currently broken)

### CODE REVIEW

**Comment by samajammin** [comment]
FYI @ctrlc03

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Description
+
+<!-- Please provide a detailed description of the pull request you are opening. -->
+
+## Additional Notes
+
+<!-- If there are any additional notes, requirements or special instructions related to this ceremony, please specify them here. -->
```
Comment (ctrlc03) [inline .github/pull_request_template.md:7]: `related to this "ceremony"` - my bad there -> please change to "PR"

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Just one comment, thank you for fixing this

File: .github/pull_request_template.md (hunk 1 above)
Comment (samajammin) [inline .github/pull_request_template.md:7]:
```suggestion
<!-- If there are any additional notes, requirements or special instructions related to this PR, please specify them here. -->
```

**Comment by samajammin** [comment]
@ctrlc03 updated 👍

**Review by ctrlc03 — APPROVED** [review]
Thank you!!

**Pull request merged**